Formula/qscintilla2.rb

```diff
@@ -11,26 +12,30 @@ class Qscintilla2 < Formula
     sha256 "a2a4e459bc424996597fdd75bd597c279e157f8818c6e106107dce516ab46c24" => :mavericks
   end
 
-  option "without-plugin", "Skip building the Qt Designer plugin"
+  option "with-qt5", "Build with Qt5 support"
+  option "with-plugin", "Build the Qt Designer plugin"
   option "without-python", "Skip building the Python bindings"
 
   depends_on :python => :recommended
   depends_on :python3 => :optional
 
-  if build.with? "python3"
-    depends_on "pyqt" => "with-python3"
-  elsif build.with? "python"
-    depends_on "pyqt"
+  if build.with?("python") || build.with?("python3")
```

Comment:
Can you split the `build.with?("python3")` into another `if`; it's a bit hard to follow this as-is.